acquisitions/src/main/resources/thunderjet/mod-gobi/features/gobi-api-tests.feature

```diff
@@ -80,3 +80,214 @@ Feature: mod-gobi api tests
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    When method GET
+    Then status 200
+    And match response.orderMappings.orderType == 'UnlistedPrintSerial'
+    And match response.orderMappings.mappings[1].dataSource.default == "true"
+    And match response.orderMappings.mappings[2].dataSource.default == "false"
+
+    # Delete non-existent custom mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    When method DELETE
+    Then status 404
+
+    # Update non-existent custom mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    And request valid_mapping
+    When method PUT
+    Then status 404
+
+    # Create custom mapping
+    Given path '/gobi/orders/custom-mappings'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    And request valid_mapping
+    When method POST
+    Then status 201
+
+    # Re-create already created custom mapping
+    Given path '/gobi/orders/custom-mappings'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    And request valid_mapping
+    When method POST
+    Then status 422
+
+    # Verify the newly added mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    When method GET
+    Then status 200
+    And match response.orderMappings.orderType == 'UnlistedPrintSerial'
+    And match response.orderMappings.mappings[1].dataSource.default == "false"
+    And match response.orderMappings.mappings[2].dataSource.default == "true"
+
+    # Update existing custom mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    And request valid_mapping
+    When method PUT
+    Then status 204
+
+    # Delete existing custom mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    When method DELETE
+    Then status 200
+
+    # Verify original mapping
+    Given path '/gobi/orders/custom-mappings/UnlistedPrintSerial'
+    And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
+    When method GET
+    Then status 200
+    And match response.orderMappings.orderType == 'UnlistedPrintSerial'
+    And match response.orderMappings.mappings[1].dataSource.default == "true"
+    And match response.orderMappings.mappings[2].dataSource.default == "false"
```

Comment:
in this and similar places we also can add assertions in these places if it custom or default to more easily distinguish mappings